test/Interop/Cxx/class/Inputs/members.h

```diff
@@ -0,0 +1,4 @@
+class MyClass {
```

Comment:
Calling the file "members" creates a really big scope. I suggest something like "member variables".

Reply:
Done